### PR TITLE
Fix ItManageNameSpace test failures.

### PR DESCRIPTION
**Pull request by sankarpn**
This PR fixes the test failures caused by replaceNameSpaceObject failures. The test will retry the labeling process and replace the namespace object until it succeeds.

In OKD

https://build.weblogick8s.org:8443/job/wko-okd-test/297
https://build.weblogick8s.org:8443/job/wko-okd-test/299

In Kind
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10271

### CODE REVIEW

**Review by robertpatrick — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java [hunk 1]
```diff
@@ -469,17 +526,15 @@ private void checkSecondOperatorFailedToShareSameNS(String domainNamespace) {
   }
 
   private void checkDomainNotStartedInDefaultNS() {
-    //verify operator can't start domain in the default namespace when domainNsSelectionStrategy not List
-    // and selector does not match default
+    logger.info("verify operator can't start domain in the default namespace when domainNsSelectionStrategy not List"
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java:529]: add space after "List"

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java [hunk 2]
```diff
@@ -363,23 +388,32 @@ void testNameSpaceWithOperatorRbacFalse() {
...
+    setTlsTerminationForRoute("external-weblogic-operator-svc", opNamespaces[3]);
     checkOperatorCanScaleDomain(opNamespaces[3], manageByLabelDomainUid);
   }
 
   private void checkUpgradeFailedToAddNSManagedByAnotherOperator() {
-    //upgrade operator1 to replace managing domains using RegExp namespaces
-    // for ns names starting from weblogic, there one of domains
-    //in namespace weblogic* is managed by operator2
+    logger.info("upgrade operator1 to replace managing domains using RegExp namespaces"
+        + " for ns names starting from weblogic, there one of domains"
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java:404]: add space after "domains"

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java [hunk 3]
```diff
@@ -284,40 +302,43 @@ void testNameSpaceManagedByLabelSelector() {
...
         + " is running in namespace " + domainNamespaces[1]);
     checkOperatorCanScaleDomain(opNamespaces[0], domainsUid[1]);
 
-    //check that with specific Selector default namespace is not under operator management
+    logger.info("check that with specific Selector default namespace is not under operator management");
     checkDomainNotStartedInDefaultNS();
 
-    //check that another operator with selector=List matching domain namespace,
-    // managed by first operator fails to install
+    logger.info("check that another operator with selector=List matching domain namespace,"
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java:317]: add space after ","

**Review by xiancao — APPROVED** [review]
minor comments

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java (hunk 3 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java]: done

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java (hunk 2 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java]: done

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNameSpace.java]: done

**Review by sankarpn — COMMENTED** [review]
done

**Pull request merged**